File: qaly.java
import java.util.*;
class qaly {
  public static void main(String[] args) {
    Scanner kbd = new Scanner(System.in);
    double tot=0;
    int N = kbd.nextInt();
    for (int i=0;i<N;i++){
      double q = kbd.nextDouble();
      double l = kbd.nextDouble();
      tot = q*l+tot;
    }
    System.out.format("%.3f", tot);
   }
}

File: qaly.cpp
#include <iostream>
using namespace std;

int main(){
    int N;
    double q;
    double l,tot;
    cin>>N;
    for (int i=0;i<N;i++){
      cin>>q;
      cin>>l;
      tot = q*l+tot;
    }
    cout<<tot;
   }